### ISSUE TITLE
How to train background inpainter?

### ISSUE THREAD

**Issue by ChennyTech** [tzt101/MichiGAN#10]
Thank you for the insightful work!

Could you provide us more details about the training process of background inpainter?
I know that you cannot provide the codes due to the copyright issue,
but I am still curious to know which training dataset and hole masks you used to train the network.
or if you just used the pretrained network, could you give us information about the company?

**Comment by thaoshibe** [comment]
hello there, have you got the answer?
